docs/introduction/naming-conventions.md

```diff
@@ -25,14 +25,36 @@ and others. You can find similar cases in the framework API which has `EventId`,
 `UserId`, `TenantId`, and others. 
 
 We find it convenient to define ID types in one file called `identifiers.proto`. 
-A typical project is likely to have more than one Bounded Context, thus you will have several
+A typical project is likely to have more than one Bounded Context. Thus, you will have several
 `identifiers.proto` files. 
 Each of them resides under the directory with proto files defining the data model of the
-corresponding Bounded Context.   
+corresponding Bounded Context. For example:
 
-### Command definitions
+```
```

Comment:
@JuliaEvseeva, I need to highlight `identifiers.proto` in this fragment. I cannot do it using the markdown syntax, which is used now.  I can do it via `<pre>`, `<code>, and `<strong>`, but it does not turn on color highlighting, which is available for other code fragments. When I put classes that are used in other code fragments (including nesting of corresponding `<div>`s) resulting page starts displaying those tags. Please suggest how I can highlight parts within a code block.

Reply:
Now we can do it like this:
```
<div class="highlighter-rouge">
<pre class="highlight">
<code>
    myproject/
      users/
        src/
          main/
            java/
            proto/
              user.proto
              group.proto
              ...
              <span class="kd">identifiers.proto</span>
      tasks/
        src/
          main/
            java/    
            proto/
              task.proto
              project.proto
              ...
              identifiers.proto
        ...
</code>
</pre>
</div>
```

Reply:
Thanks. It worked. Strangely, it didn't work for me when I tried the same. Presumably, because of the wrong nesting of tags.

Reply:
Unfortunately, this method will not work if we want to define a language for the code. I'll think about how to make it more universal.